CHANGES-4.3.md

```diff
@@ -23,6 +23,8 @@ File format:
   `SELECT topic =~ 't' as r FROM "$events/client_connected"`.
   The topic is a null value as there's no such field in event `$events/client_connected`, so it
   should return false if match it to a topic.
+- Added a test to prevent a last will testament message to be
```

Comment:
this is more than adding a test case?
we also fixed some potential issues?
e.g. will not send will message when takeover happens ?